### PR TITLE
feat(structure): create an example Unity project

### PR DESCRIPTION
**Pull request by extendreality**
The VRTK repo is now simply an example Unity project that includes the
relevant dependencies of the VRTK prefabs to showcase how they can
be used to build something.

It removes the ability to add VRTK to a project via a package.json as
this is now expected to be done by adding the VRTK.Prefabs repo.

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  ],
+  "dependencies": {
+    "com.unity.ads": "2.0.8",
+    "com.unity.analytics": "3.2.2",
+    "com.unity.collab-proxy": "1.2.15",
+    "com.unity.package-manager-ui": "2.0.7",
+    "com.unity.purchasing": "2.0.3",
+    "com.unity.textmeshpro": "1.3.0",
+    "com.unity.xr.oculus.standalone": "1.38.3",
+    "io.extendreality.vrtk.prefabs": "1.0.1",
```
Comment (bddckr) [inline Packages/manifest.json]: Move this up in `dependencies` so it's the first entry instead.

File: README.md [hunk 2]
```diff
@@ -18,69 +19,20 @@ VRTK v4 has not yet been released fully is still in the beta phase meaning there
...
-    "dependencies": {
-      "io.extendreality.vrtk": "X.Y.Z",
-      ...
-    }
-  }
-  ```
-* Switch back to the Unity software and wait for it to finish importing the added package.
+### Downloading the project
+
+* Download this project repository to your local machine using one of the following methods:
```
Comment (bddckr) [inline README.md]:
```suggestion
* Download this project repository to your local machine using *one* of the following methods:
```

File: README.md [hunk 3]
```diff
@@ -18,69 +19,20 @@ VRTK v4 has not yet been released fully is still in the beta phase meaning there
...
+  * Git clone the repository with `git clone https://github.com/ExtendRealityLtd/VRTK.git`
+  * or simply download the zip file at `https://github.com/ExtendRealityLtd/VRTK/archive/master.zip` and extract to your local machine.
```
Comment (bddckr) [inline README.md]:
```suggestion
  * Download the zip file at `https://github.com/ExtendRealityLtd/VRTK/archive/master.zip` and extract it.
```

File: Packages/manifest.json (hunk 1 above)
Comment (thestonefox) [reply Packages/manifest.json]: Unity auto changes this, I did put it first and unity swapped it

Comment (thestonefox) [reply Packages/manifest.json]: i changed it anyway, but nothing to say unity wont swap it again for other people.

**Pull request merged**